src/server/language.py

```diff
@@ -48,6 +48,7 @@ class Language(object):
     FR = _Language('Français', 'fr')
     HI = _Language('हिन्दी', 'hi')
     JA = _Language('日本語', 'ja')
+    NL = _Language('Dutch', 'nl')
```

Comment:
This should be in local language so shouldn't it be Nederlands?

```suggestion
    NL = _Language('Nederlands', 'nl')
```